src/test/java/no/sikt/sws/testutils/TestUtils.java

```diff
@@ -57,4 +58,8 @@ public static  <T> InputStream buildRequestWithBody(HttpMethod httpMethod,
                 .withBody(body)
                 .build();
     }
+
+    public static String readCompact(String body) {
```

Comment:
you can postpone it to another PR, but we should make sure this function is used in all tests, instead of the local functions which do the same

Reply:
Also a better name could be something like "stripWhitespace"